src/libs/kata-types/src/mount.rs

```diff
@@ -68,6 +68,19 @@ pub fn is_kata_host_dir_volume(ty: &str) -> bool {
     ty == KATA_HOST_DIR_VOLUME_TYPE
 }
 
+/// nydus extraoptions
+#[derive(Debug, serde::Deserialize)]
+pub struct NydusExtraOption {
```

Comment:
I think it's better to rename it to `NydusExtraOptions`.

Reply:
fixed